core/domain/stats_services.py

```diff
@@ -119,50 +101,14 @@ def _get_explorations_states_tuples_by_ids(exp_ids):
     for exp_id in exps_answers_mapping:
         exps_answers_mapping[exp_id]['unresolved_answers'] = (sorted(
             exps_answers_mapping[exp_id]['unresolved_answers'],
-            key=lambda a: a['count'],
+            key=lambda a: a['frequency'],
             reverse=True))
 
     return exps_answers_mapping
 
 
-def get_state_rules_stats(exploration_id, state_name):
-    """Gets statistics for the answer groups and rules of this state.
-
-    Returns:
-        A dict, keyed by the string '{HANDLER_NAME}.{RULE_STR}', whose
-        values are the corresponding stats_domain.StateRuleAnswerLog
-        instances.
-    """
-    exploration = exp_services.get_exploration_by_id(exploration_id)
-    state = exploration.states[state_name]
-
-    rule_keys = []
-    for group in state.interaction.answer_groups:
-        for rule in group.rule_specs:
-            rule_keys.append((
-                _OLD_SUBMIT_HANDLER_NAME, rule.stringify_classified_rule()))
-
-    if state.interaction.default_outcome:
-        rule_keys.append((
-            _OLD_SUBMIT_HANDLER_NAME, exp_domain.DEFAULT_RULESPEC_STR))
-
-    answer_logs = stats_domain.StateRuleAnswerLog.get_multi(
-        exploration_id, [{
-            'state_name': state_name,
-            'rule_str': rule_key[1]
-        } for rule_key in rule_keys])
-
-    results = {}
-    for ind, answer_log in enumerate(answer_logs):
-        results['.'.join(rule_keys[ind])] = {
-            'answers': answer_log.get_top_answers(5),
-            'rule_hits': answer_log.total_answer_count
-        }
-
-    return results
-
-
-def get_visualizations_info(exploration_id, exploration_version, state_name):
+# TODO(bhenning): Test.
```

Comment:
Agreed, but I'm inclined to do it in a later PR. Happy to file an issue if you agree (I heavily modified this code, but it was likely originally authored by @marcel-sl).